webpack.client.common.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Pretty much the same as [what we used to have](https://github.com/mozilla/network-pulse/blob/master/webpack.config.js). I removed plugin related code that's no longer need based the [migration guide](https://webpack.js.org/migrate/4/#deprecatedremoved-plugins).